VK_V1.0/VK_V1.0/Animations/cloudView.swift

```diff
@@ -0,0 +1,127 @@
+// cloudView.swift
```

Comment:
```suggestion
// CloudView.swift
```
и файлик лучше тоже переименовать с большой буквы